CONTRIBUTING.md

```diff
@@ -130,41 +122,51 @@ Issue #190: Short summary of the issue
 ```
 
 Code
-====
+----
 
 This section include some tips that will help you to push source code.
 
-::: {.note}
-::: {.admonition-title}
-Note
-:::
+**Note:**
+> We recommend using a self-contained environment, 
+> such as [Python virtual environments](https://docs.python.org/3/tutorial/venv.html) 
+> or Anaconda environments, to cleanly separate Qiskit from other applications 
```

Comment:
Can you add a link to Anaconda? Preferably something that talks about setting up a virtual env (similar to the python vir env link)